dataactbroker/fileRoutes.py

```diff
@@ -130,6 +130,70 @@ def get_protected_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.get_protected_files()
 
+    @app.route("/v1/check_current_page/", methods=["GET"])
+    @convert_to_submission_id
```

Comment:
Do we not want to check permissions for the submission in question? If a user with no permissions accesses this route, they would still know what phase the submission is in.